dev-docs/RFCs/v7.x/spring-based-transitions-rfc.md

```diff
@@ -7,11 +7,40 @@
 
 ## Abstract
 
-[FILL ME IN]
+As previous animation- and transition-related RFCs have pointed out, easy-to-use animation and
```

Comment:
Usually we keep an abstract to 2-3 lines just summarizing what is being proposed so people can scan multiple RFCs quickly, and then we have separate background section of about the length you have in here.

Reply:
Oh okay! Will update!